pilot/pkg/config/kube/crd/controller/client.go

```diff
@@ -84,6 +84,8 @@ type restClientBuilder struct {
 	types          []*crd.SchemaType
 }
 
+var scope = log.RegisterScope("client", "", 0)
```

Comment:
`kube` maybe? client is pretty generic and not what I would expect to enable for kube logs.

Reply:
Now using 'kube'.